src/config/consts.js

```diff
@@ -43,6 +43,10 @@ export const consts = {
     infoProvider: 'Anbieterinformationen (Taste)',
     lastName: 'Nachname',
     link: '(Link)',
+    login: 'Anmelden (Taste)',
+    loginHint: 'Zu den Anmeldedaten wechseln',
+    logout: 'Abmelden (Taste)',
+    logoutHint: 'Zu den Anmeldedaten wechseln',
```

Comment:
is it intended to habe the same texts for `loginHint` and `logoutHint`? and where does `Zu den Anmeldedaten wechseln` come from? i don't get the meaning correctly, what should it tell someone?

Reply:
Yes, you are right. loginHint and logoutHint should not be the same. These texts were added here for accessibility. If the user is logged in, the device reads logoutHint to the user.